libraries/standard/http/test/common.h

```diff
@@ -7,4 +7,8 @@
 #include "private/http_client_internal.h"
 #include "private/http_client_parse.h"
 
-#define assert( x )
+#if TESTS_DISABLE_ASSERT
```

Comment:
let's do this instead and then we can add tests for our asserts later
```
static int assertCount_;
#define assertReset()	do { assertCount_ = 0; } while( 0 )
```

Reply:
Sure, I prefer that too.

Reply:
Please stick to one variable naming either _xyzAbc or xyzAbc_ to avoid confusion.

Reply:
Good point. Will change to `_assertCount`

Reply:
Updated